drivers/soc/samsung/Kconfig

```diff
@@ -200,6 +200,9 @@ config EXYNOS_SECURE_LOG
 	help
 	  Support Exynos Secure Log
 
+config STATE_NOTIFIER
+        bool "State Notifier"
```

Comment:
What is this driver for? It looks like it's "helper" for other drivers. However, I can't see drivers that use this one.

Reply:
For some governors and io schedulers.

Reply:
In fact maple and fiops are working better with this.

Reply:
Current maple, fiops implementations doesn't call state notifier. Maybe there are other maple, fiops implementations

Reply:
My bad, 3.18 kernel does that.